app/controllers/CiapiController/CiapiController.scala

```diff
@@ -42,4 +42,12 @@ class CiapiController @Inject()(appConfig: AppConfig,
       Future.successful(NotFound)
     }
   }
+
+	def vatOnline: Action[AnyContent] = Action.async { implicit request =>
+		if(config.showVATCUI) {
+			Future.successful(Ok(vatOnlineCuiView()))
```

Comment:
Should `vatOnlineCuiView` be imported and added to the constructor for this controller? It's not compiling for me